File: uri/calculosimples/calculosimples.rb
def calculosimples (npecasA, valorA, npecasB, valorB)
    return ( Float( npecasA ) * Float( valorA) ) +  (  Float( npecasB ) * Float( valorB) )
end

linha1 = gets.chomp
linha2 = gets.chomp
varA = linha1.split(" ")
varB = linha2.split(" ")

resultado = calculosimples( varA[1], varA[2], varB[1], varB[2])
puts "VALOR A PAGAR: R$ %.2f" % [ resultado ] 

File: uri/calculosimples/calculosimples.py
# ****************************************************************
"""
def calculosimples (npecasA, valorA, npecasB, valorB):
    return ( float( npecasA ) * float( valorA) ) +  (  float( npecasB ) * float( valorB) )

codeA, npecasA, valorA = raw_input().split()
codeB, npecasB, valorB = raw_input().split()
resultado = calculosimples( npecasA, valorA, npecasB, valorB)
print "VALOR A PAGAR: R$ {0:.2f}". format(resultado) 

"""

#**********************************************************************

def calculosimples (npecasA, valorA, npecasB, valorB):
    return ( float( npecasA ) * float( valorA) ) +  (  float( npecasB ) * float( valorB) )

codeA, npecasA, valorA = input().split()
codeB, npecasB, valorB = input().split()
resultado = calculosimples( npecasA, valorA, npecasB, valorB)
print ("VALOR A PAGAR: R$ {0:.2f}". format(resultado))
